wahoomc/osm_maps_functions.py

```diff
@@ -40,30 +40,24 @@ def run_subprocess_and_log_output(cmd, error_message, cwd=""):
     run given cmd-subprocess and issue error message if wished
     """
     if not cwd:
-        process = subprocess.Popen(
-            cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+        process = subprocess.run(
+            cmd, capture_output=True, text=True, encoding="utf-8")
```

Comment:
```suggestion
            cmd, capture_output=True, text=True, encoding="utf-8", check=False)
```

pylint suggests to give `check` parameter. Default is False. I think False would be a good value so that no exception is raised, because we handle giving output by ourself. I am not 100% sure, what do you think? See https://docs.python.org/3.5/library/subprocess.html#subprocess.run